### ISSUE TITLE
[Bug]Error using the UMD file kepler.min.js

### ISSUE THREAD

**Issue by geostarters** [keplergl/kepler.gl#1034]
**Error using the UMD file kepler.min.js**

I have cloned and installed the project correctly but when I try to use the generated local file **/umd/keplergl.min.js** with the example of **/umd-client/index.html** `

```html
 <!-- Load Kepler.gl-->
  <!-- <script src="https://unpkg.com/kepler.gl@2.0.1/umd/keplergl.min.js"></script>-->
  <!--If you want to load the build from filepath -->
  <script src="../../umd/keplergl.min.js"></script>
 ```  
 the browser console   error returns this error
```
Uncaught TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Object'
    |     property 'Provider' -> object with constructor 'Object'
    --- property '_context' closes the circle
    at JSON.stringify (<anonymous>)
    at react-redux.min.js:1
    at keplergl.min.js:55
    at G (keplergl.min.js:38)
    at e (keplergl.min.js:38)
    at keplergl.min.js:38
    at Array.map (<anonymous>)
    at Object.e [as get] (keplergl.min.js:38)
    at Object.<anonymous> (keplergl.min.js:55)
    at n (keplergl.min.js:1)
index.html:58 Uncaught ReferenceError: KeplerGl is not defined
    at index.html:58
index.html:124 Uncaught ReferenceError: KeplerGl is not defined
    at index.html:124
```

**Screenshots**
![image](https://user-images.githubusercontent.com/7668442/77685250-7d3d3a00-6f9b-11ea-843a-9a27418edaab.png)

**Desktop:**
 - OS: Win10 and Ubuntu18
 - Browser [e.g. chrome, brave]
 
**Additional context**
Yarn 1.22.4
Node 12.16.1
nmp:6.13.4

**Comment by heshan0131** [comment]
@geostarters Are you building your own `keplergl.min.js`? Looks like there is an error in your our own keplergl.min.js

**Comment by geostarters** [comment]
Yes, that's our intention, but so far we've only "fresh" compiled it and haven't started to develop it yet.

**Comment by heshan0131** [comment]
https://github.com/keplergl/kepler.gl/pull/1041

**Comment by heshan0131** [comment]
we are deploying a fix, and cutting a new release 2.1.1

**Comment by geostarters** [comment]
Now it works! Thank you for your prompt resolution... Much appreciated!
